include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -481,4 +481,12 @@ SECTIONS
 	KEEP(*(.gnu.attributes))
 	}
 
+SECTION_PROLOGUE(.last_section,(NOLOAD),)
```

Comment:
Curious here.
Can we create empty sections and ask it to give the load address when there is no data to load?

Reply:
Not sure what the specification says about this, but it works as I intended.
The empty section will still be placed in the output, with zero length.
Hence we can use its load address.

Last lines in .map file:
```
last_section 0xb3f0 0x0
             0xb3f0    _flash_used = (LOADADDR (.last_section) - _image_rom_start)
```

The